pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -513,43 +509,47 @@ func (b *CPUBurst) applyCPUBurst(burstCfg *slov1alpha1.CPUBurstConfig, podMeta *
-					"updated %v, info %v", pod.Namespace, pod.Name, containerStat.Name, containerDir, updated, err)
-			}
+		podCFSBurstVal += containerCFSBurstVal
+		containerCFSBurstValStr := strconv.FormatInt(containerCFSBurstVal, 10)
+		updater, err := resourceexecutor.GetCgroupUpdater(system.CPUBurstName, containerDir, containerCFSBurstValStr)
+		if err != nil {
+			klog.V(4).Infof("get cpu burst updater for container %s/%s/%s failed, err: %v",
+				pod.Namespace, pod.Name, containerStat.Name, err)
+			continue
 		}
-	} // end for containers
-
-	podDir := koordletutil.GetPodCgroupDirWithKube(podMeta.CgroupDir)
-	if system.ValidateResourceValue(&podCFSBurstVal, podDir, system.CPUBurst) {
-		ownerRef := executor.PodOwnerRef(pod.Namespace, pod.Name)
-		podCFSBurstValStr := strconv.FormatInt(podCFSBurstVal, 10)
-		updater := executor.NewCommonCgroupResourceUpdater(ownerRef, podDir, system.CPUBurst, podCFSBurstValStr)
-		updated, err := b.executor.UpdateByCache(updater)
+		updated, err := b.executor.Update(true, updater)
 		if err == nil {
-			klog.V(5).Infof("apply pod %v/%v cpu burst value success, dir %v, value %v",
-				pod.Namespace, pod.Name, podDir, podCFSBurstValStr)
+			klog.V(5).Infof("apply container %v/%v/%v cpu burst value success, dir %v, value %v",
+				pod.Namespace, pod.Name, containerStat.Name, containerDir, containerCFSBurstVal)
 		} else if system.HostSystemInfo.IsAnolisOS {
 			// cgroup `cpu.burst_us` is expected available on anolis os, and it may not exist in other kernels.
-			klog.Infof("update pod %v/%v cpu burst failed, dir %v, updated %v, error %v",
-				pod.Namespace, pod.Name, podDir, updated, err)
+			klog.Infof("update container %v/%v/%v cpu burst failed, dir %v, updated %v, error %v",
+				pod.Namespace, pod.Name, containerStat.Name, containerDir, updated, err)
 		} else {
-			klog.V(4).Infof("update pod %v/%v cpu burst ignored on non Anolis OS, dir %v, updated %v, "+
-				"info %v", pod.Namespace, pod.Name, podDir, updated, err)
+			klog.V(4).Infof("update container %v/%v/%v cpu burst ignored on non Anolis OS, dir %v, "+
+				"updated %v, info %v", pod.Namespace, pod.Name, containerStat.Name, containerDir, updated, err)
 		}
+	} // end for containers
+
+	podDir := koordletutil.GetPodCgroupDirWithKube(podMeta.CgroupDir)
+	podCFSBurstValStr := strconv.FormatInt(podCFSBurstVal, 10)
+	updater, err := resourceexecutor.GetCgroupUpdater(system.CPUBurstName, podDir, podCFSBurstValStr)
+	if err != nil {
+		klog.V(4).Infof("get cpu burst updater for pod %s/%s failed, err: %v",
+			pod.Namespace, pod.Name, err)
+		return
+	}
+	updated, err := b.executor.Update(true, updater)
+	if err == nil {
+		klog.V(5).Infof("apply pod %v/%v cpu burst value success, dir %v, value %v",
+			pod.Namespace, pod.Name, podDir, podCFSBurstValStr)
+	} else if system.HostSystemInfo.IsAnolisOS {
+		// cgroup `cpu.burst_us` is expected available on anolis os, and it may not exist in other kernels.
```

Comment:
IsAnolisOS is not needed since each cgroup file has its own validator

Reply:
fixed